database_cleanup/models/purge_wizard.py

```diff
@@ -25,7 +25,7 @@ class CleanupPurgeLine(models.AbstractModel):
     def purge(self):
         raise NotImplementedError
 
-    @api.model
+    @api.model_create_multi
     def create(self, values):
```

Comment:
```suggestion
    def create(self, vals_list):
```

nitpicking : by convention, AFAIK, the argument is vals_list when we use the decorator model_create_multi.

(same remarks below).
otherwise, LGTM.

Reply:
Yeah, I totally agree with you

The reason I didn't change it is because in 17.0 it's done exactly like this, without renaming the arguments. And so I figured this way there's absolutely nothing to forward-port.

But I'm not sure my reasoning is correct. I can change it if you think it's better

Reply:
If it's not upstream, it doesnt matter for me. Improvment should ne done on latest révision.